composer.json

```diff
@@ -154,7 +154,7 @@
         "tracy/tracy": "^2.4.13",
         "trikoder/oauth2-bundle": "^1.1",
         "twig/extensions": "^1.5.1",
-        "twig/twig": "^2.4.8",
+        "twig/twig": "^2.9",
```

Comment:
What about version 3?

Reply:
Twig ^3 required php 7.2.5, but I updated to min version 2.12 - c9377116af84515890342074d974f15a4c4c7306